kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,15 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangoWhiteNoise):
+
+    # Custom MIME types
+    KUMA_MIMETYPES = (
+        ('application/font-woff', '.woff'),
+        ('application/font-woff2', '.woff2'),)
```

Comment:
Whitenoise uses `font/woff2` as the mimetype for that file extension, that's no good?